utility/start_server.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# running django database migrations 
+python /app/manage.py migrate
+# creating test data if local settings are configured
+if [ "${DJANGO_SETTINGS_MODULE}" = "config.settings.local" ] ; then
+    echo "CREATING ADMIN USER FOR TESTING PURPOSES..."
+    # Using key error as an indicator for whether or not to run database population and job scheduling scripts
+    echo "from django.contrib.auth import get_user_model; User = get_user_model(); User.objects.create_superuser('admin', 'admin@salesforce.com', 'password')" | python manage.py shell
+    if [ $? -eq 0 ] ; then
+        # populating database with test repository, done only once
+        echo "POPULATING DATABASE WITH TEST DATA..."
+        python /app/manage.py populate_db;
+        # running job scheduler 
+        python /app/manage.py metaci_scheduled_jobs;
+        npm rebuild node-sass
+    else
+        # Redirect stdout from echo command to stderr.
+        echo "Admin user has already been created."
+    fi
+fi
+# creating for authorization
+echo $SFDX_HUB_KEY > /app/sfdx_hub.key
```

Comment:
When running in VSCode this file gets synced back to the host and shows as an untracked file for git. Please add it to .gitignore